src/modules/main/sidebar/Sidebar.tsx

```diff
@@ -55,9 +69,17 @@ export const Sidebar = ({ basePath }: SidebarProps) => {
               to={`${basePath}${path}`}
             >
               <Button>
-                {icon}
-                <span>{label}</span>
+
+                <div style={{
```

Comment:
Avoid inline css

Reply:
Inline styles have been replaced with styles in the .scss file